Simplenote/Classes/SPNoteListViewController.m

```diff
@@ -218,13 +223,10 @@ - (void)refreshStyle {
 }
 
 - (void)updateNavigationBar {
-    if (tagFilterType == SPTagFilterTypeDeleted) {
-        [self.navigationItem setRightBarButtonItem:_emptyTrashButton animated:YES];
-        [self.navigationItem setLeftBarButtonItem:_sidebarButton animated:YES];
-    } else {
-        [self.navigationItem setRightBarButtonItem:_addButton animated:YES];
-        [self.navigationItem setLeftBarButtonItem:_sidebarButton animated:YES];
-    }
+    UIBarButtonItem *rightButton = (self.tagFilterType == SPTagFilterTypeDeleted) ? _emptyTrashButton : _addButton;
+
+    [self.navigationItem setRightBarButtonItem:rightButton animated:YES];
+    [self.navigationItem setLeftBarButtonItem:_sidebarButton animated:YES];
```

Comment:
Maybe use `self.` vs referencing the ivars directly?

Reply:
Thank you!!!